README.md

```diff
@@ -190,7 +190,7 @@ see below for a full list.
 
 ```ignore
 USAGE:
-    lychee [FLAGS] [OPTIONS] [--] [inputs]...
+    lychee [FLAGS] [OPTIONS] <inputs>...
```

Comment:
Probably keep that `[--]`? I mean the hint is in mentioned below as well, but I remember I've overseen it in one place and required a few attempts before I figured out that `--` was possible and required to have the last arguments count as inputs instead of option arguments 😄.

Reply:
Yeah I noticed that as well, but I ran some tests and couldn't find a case where the `--` was still required. Structopt seems to remove the `[--]` from the usage section if it's not strictly necessary. 🤷‍♀️ 
Any way you can reproduce the old failing behavior with this branch? If so, we need to find out how to enforce the old syntax in the usage info (which is auto-generated and tested by a doc-test, so we can't just manually edit it). If not, we might want to remove the hint below. (Which I count as a win. 😄)

BTW this also still works, so it's not a breaking change unless we can find another issue:

```
lychee -- README.md
```

Reply:
Ah, even when using an option with multiple value support, like:
```sh
lychee --exclude 'this' 'that' 'next' 'README.md'
```
How does it know where the excludes end and where the inputs start?

Reply:
This is the code for the `[--]` part:
https://github.com/clap-rs/clap/blob/88dec14775fc94170b79c38d88da5385c5b9b6e8/src/output/usage.rs#L84-L106

Reply:
So the easiest thing was to add `last = true`. No idea if that's how it's meant to be done. 😆

Reply:
So now the last input argument is automatically used as input in such case? Still good to keep the hint visible as also this may happen:
```sh
lychee --exclude 'this' 'that' index.html README.md
```

Reply:
I understood it as "inputs need to be the last arguments now".

Reply:
Okay, then I don't understand how `last = true` helps in this context 😄. However, I think its okay when `--` is required in some cases. Better to force users to be explicit than being implicit and guess wrong about what was meant. The only other alternative I see would be to allow only single options values, and e.g. have multiple excludes/schemes/etc separated by `,`. But especially with excludes which may contain literal `,` themselves it is a bit difficult and would be a breaking change.

... although also multiple excludes (same with includes) can be merged into a single regex (we do it like that):
```
lychee --exclude '^https://(example.org|another.com|localhost)'
```
and multiple schemes can easily be comma-separated. Only with headers it would be complicated to achieve.

Very interesting that the `clap-rs` crate adds `[--]` when there are options which allow multiple values (the code look quite clear about that) and even that there are several which do, it didn't add it in our case?

Reply:
@mre 
> BTW this also still works, so it's not a breaking change unless we can find another issue:

@MichaIng 
> Very interesting that the clap-rs crate adds [--] when there are options which allow multiple values (the code look quite clear about that) and even that there are several which do, it didn't add it in our case?


It's a convention that `--` signals the end of CLI options/flags. Anything after that is parsed as positional arguments.

But it's still helpful to remind users of `--`. Suppose a filename starts with `-`, one way to handle it correctly is to use `lychee -- -FILENAME`.